Portfolio/index.html

```diff
@@ -1,209 +1,236 @@
+                        experiences.
+                        With over 2 years of hands-on experience, i bring a deep proficiency in HTML, CSS, JavaScript,
+                        TypeScript, and a keen mastery of modern frameworks like React.js, React Native, Svelte.js. I
+                        specialize in building responsive, interactive, and visually appealing user interfaces.
+                        Additionally, my proficiency in Firebase ensures secure and efficient backend integration,
+                        making me
+                        a versatile and dynamic developer ready to tackle any project</strong>
+                </p>
+                <h4>
+                    <div><strong> Name:John Doe</strong> </div>
+                    <div><strong>Age:24</strong> </div>
+                    <div><strong> Experience: 2years</strong> </div>
+                </h4>
+            </div>
         </div>
     </section>
 
     <!-- skills -->
-    <section class="skills" id="skills">
-
-        <h1>SKILLS</h1>
 
-        <h4> <img src="/html.icon.png" alt="Css Image" width="30px" height="30px">
-            <div class="skilltext">HTML</div>
-        </h4>
+    <section class="myskills">
 
-        <h4> <img src="/css.icon.png" alt="Css Image" width="30px" height="30px">
-            <div class="skilltext">CSS</div>
-        </h4>
+        <h1>Skills</h1>
+        <div class="containerss">
 
-        <h4> <img src="/js.icon.png" alt="Css Image" width="30px" height="30px">
-            <div class="skilltext">JavaScript</div>
-        </h4>
 
-        <h4> <img src="/ts.icon.png" alt="Css Image" width="30px" height="30px">
-            <div class="skilltext">TypeScript</div>
-        </h4>
+            <div class="image-container">
+                <img src="/html.icon.png" alt="Sample Image" width="30px" height="30px">
+                <h2>HTML</h2>
+            </div>
 
-        <h4> <img src="/svelte.icon.png" alt="Css Image" width="30px" height="30px">
-            <div class="skilltext">Svelte.js</div>
-        </h4>
+            <div class="image-container">
+                <img src="/css.icon.png" alt="Sample Image" width="30px" height="30px">
+                <h2>CSS</h2>
+            </div>
+            <div class="image-container">
+                <img src="/js.icon.png" alt="Sample Image" width="30px" height="30px">
+                <h2>JavaScript</h2>
+            </div>
+            <div class="image-container">
+                <img src="/ts.icon.png" alt="Sample Image" width="30px" height="30px">
+                <h2>TypeScript</h2>
+            </div>
+            <div class="image-container">
+                <img src="/svelte.icon.png" alt="Sample Image" width="30px" height="30px">
+                <h2>Svelte</h2>
+            </div>
+            <div class="image-container">
+                <img src="/firebase.icon.png" alt="Sample Image" width="20px" height="20px">

```

Comment:
The 'justify-content' and 'align-items' properties should not be applied within the .containers h3. These properties should be defined at the container level.